### DISCUSSION TITLE
Email Previously Verified Now Not Recognized

### DISCUSSION THREAD

**Discussion by MarcoSpanoDev** [community/community#121201] [GitHub Education] [answered]
### Hi everyone,

In 2022, I used this email to verify my student status, and it worked. Now, it requires a student email in my account, which I have. But it says that I don't have a student email. Contacting support is almost impossible because I am always directed to reach out to the community here. Does anyone know how I can resolve this issue or get in direct contact with support?

**Comment by samus-aran** [comment, marked as answer]
Hi @Clytax 

Can I suggest you change your verified email to being all lower case. That is the only problem that I can anticipate. Let me know how you get on after updating your school email on your profile and you reapply.

**Reply by MarcoSpanoDev** [reply]
That did it. Thanks a lot!
Still kinda weird that it worked before
